python/dgl/graphbolt/impl/torch_based_feature_store.py

```diff
@@ -560,28 +564,41 @@ class TorchBasedFeatureStore(BasicFeatureStore):
     ...     gb.OnDiskFeatureData(domain="node", type="paper", name="feat",
     ...         format="numpy", path="/tmp/node_feat.npy", in_memory=False),
     ... ]
-    >>> feature_sotre = gb.TorchBasedFeatureStore(feat_data)
+    >>> feature_store = gb.TorchBasedFeatureStore(feat_data)
     """
 
-    def __init__(self, feat_data: List[OnDiskFeatureData]):
+    def __init__(
+        self,
+        feat_data: List[OnDiskFeatureData],
+        disk_based_feature_list: Set[FeatureKey] = None,
```

Comment:
set or list?

Reply:
Dropped the list suffix from the parameter names.